pages/windows/if.md

```diff
@@ -0,0 +1,32 @@
+# if
+
+> Performs conditional processing in batch scripts.
+> More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/if>.
+
+- Execute the specified commands if the condition is true:
+
+`if {{condition}} (echo Condition is true)`
```

Comment:
Should the echo command use double quotes to surround the thing that is being echoed?

Reply:
No, because it treat them literally.